resolver/SuggestPackage.cc

```diff
@@ -57,8 +57,11 @@ class PackageContext final {
             fmt::map_join(srcPkg.fullName(), "::", [&](auto nr) -> string { return nr.show(ctx); })));
         lines.emplace_back(
             core::ErrorLine::from(match.symbol.loc(ctx), "Constant `{}` is defined here:", match.symbol.show(ctx)));
-        // TODO(nroman-stripe) Add automatic fixers
         e.addErrorSection(core::ErrorSection(lines));
+
+        if (auto autocorrect = srcPkg.addExport(ctx, match.symbol, false)) {
+            e.addAutocorrect(std::forward<core::AutocorrectSuggestion>(*autocorrect));
```

Comment:
ooc, what's the `std::forward` for?

Reply:
The error without it is
```
resolver/SuggestPackage.cc:78:30: error: rvalue reference to type 'sorbet::core::AutocorrectSuggestion' cannot bind to lvalue of type 'std::optional<sorbet::core::AutocorrectSuggestion>::value_type' (aka 'sorbet::core::AutocorrectSuggestion')
            e.addAutocorrect(*autocorrect);
                             ^~~~~~~~~~~~
./core/Error.h:162:49: note: passing argument to parameter 'autocorrect' here
    void addAutocorrect(AutocorrectSuggestion &&autocorrect);
                                                ^
```
which happens because our `ErrorBuilder` has the method declaration
```c++
void addAutocorrect(AutocorrectSuggestion &&autocorrect);
```
so we need `std::forward` to turn the lvalue into an rvalue. I suppose I could use `std::move` instead, if we'd prefer.

Reply:
yeah i'd prefer `move` i think. thanks!

Reply:
Replaced with `move`!